internal/utils/utils.go

```diff
@@ -73,7 +73,8 @@ func ConvertFixedDataIntoByteArray(data interface{}) []byte {
 func GetUniqueIDFromIPPort(ip, port string) uint32 {
 	reg, err := regexp.Compile("[^0-9]+")
 	if err != nil {
-		log.Panic("Regex Compilation Failed", "err", err)
+		GetLogger().Crit("Regex Compilation Failed", "err", err)
+		panic(err)
```

Comment:
same thing, no need to panic here.

Reply:
If we are certain that this regex is always going to compile, can we skip the error check completely?

Reply:
agree, not need to check the error here with the simple regex.